src/Vulkan/Silk.NET.Vulkan/Enums/VideoCodecOperationFlagsKHR.gen.cs

```diff
@@ -15,15 +15,30 @@ public enum VideoCodecOperationFlagsKHR : int
     {
         [NativeName("Name", "")]
         None = 0,
+        [Obsolete("Deprecated in favour of \"InvalidBitKhr\"")]
         [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_INVALID_BIT_KHR")]
         VideoCodecOperationInvalidBitKhr = 0,
+        [Obsolete("Deprecated in favour of \"EncodeH264BitExt\"")]
         [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_ENCODE_H264_BIT_EXT")]
         VideoCodecOperationEncodeH264BitExt = 65536,
+        [Obsolete("Deprecated in favour of \"EncodeH265BitExt\"")]
         [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_ENCODE_H265_BIT_EXT")]
         VideoCodecOperationEncodeH265BitExt = 131072,
+        [Obsolete("Deprecated in favour of \"DecodeH264BitExt\"")]
         [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_DECODE_H264_BIT_EXT")]
         VideoCodecOperationDecodeH264BitExt = 1,
+        [Obsolete("Deprecated in favour of \"DecodeH265BitExt\"")]
         [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_DECODE_H265_BIT_EXT")]
         VideoCodecOperationDecodeH265BitExt = 2,
+        [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_INVALID_BIT_KHR")]
+        InvalidBitKhr = 0,
+        [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_ENCODE_H264_BIT_EXT")]
+        EncodeH264BitExt = 65536,
+        [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_ENCODE_H265_BIT_EXT")]
+        EncodeH265BitExt = 131072,
+        [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_DECODE_H264_BIT_EXT")]
+        DecodeH264BitExt = 1,
+        [NativeName("Name", "VK_VIDEO_CODEC_OPERATION_DECODE_H265_BIT_EXT")]
+        DecodeH265BitExt = 2,
```

Comment:
So sorry that I'm noticing this only now.
Shouldn't these be without the "BitKhr/Ext" suffixes?

Reply:
Ok, if the bit is consistent and not ment to be excluded (and isn't in other places) I'm happy to merge this, just not too sure what the intended rules are here

Reply:
Can confidently say that this is as intended :D